packages/devtools_app/lib/src/shared/offline_data.dart

```diff
@@ -131,13 +129,6 @@ mixin OfflineScreenControllerMixin<T> on AutoDisposeControllerMixin {
   /// included in the offline data snapshot for this screen.
   OfflineScreenData prepareOfflineScreenData();
 
-  /// Defines how the offline data for this screen should be processed and set.
-  ///
-  /// Each screen controller that mixes in [OfflineScreenControllerMixin] is
-  /// responsible for setting up the data models and feeding the data to the
-  /// screen for offline viewing - that should occur in this method.
```

Comment:
can you move this dart doc to the `maybeLoadOfflineData` method's dart doc to describe the new param?